modules/labs-react/pagination/lib/Pagination/GoTo/TextInput.tsx

```diff
@@ -4,6 +4,7 @@ import {TextInput, TextInputProps} from '@workday/canvas-kit-react/text-input';
 
 export type GoToTextInputProps = TextInputProps & {
   'aria-label': string;
+  value?: string | number;
```

Comment:
It's related to the changes in this PR, specifically the use of `createComponent` for `TextInput`. If you look at [the changes to `TextInputProps`](https://github.com/Workday/canvas-kit/pull/1068/files#diff-de6c3686ab8bc1cfd55f609d7988122166a7249cc92fa00829fc91668d2243c1R13) in this PR, you'll see that it no longer extends from `React.InputHTMLAttributes<HTMLInputElement>`. This is because that portion of the `TextInput` interface (the HTML attributes of the underlying element being represented by `TextInput`) is now being handled by `createComponent` rather than being specified in the `TextInputProps` definition.

Without `value` in `GoToTextInputProps`, TS complains about `value` not existing in `GoToTextInputProps` when it's referenced later in this file (because `value` is no longer in `TextInputProps`).